src/main/java/codeu/model/store/basic/ConversationStore.java

```diff
@@ -113,6 +116,34 @@ public Conversation getConversationWithTitle(String title) {
     return null;
   }
 
+  /**
+   * Access the Conversation object with the given UUID.
+   *
+   * @return null if the UUID does not match any existing Conversation.
+   */
+  public Conversation getConversation(UUID id) {
+    for (Conversation conversation : conversations) {
+      if (conversation.getId().equals(id)) {
+        return conversation;
+      }
+    }
+    return null;
+  }
+
+  /** Access the conversations created within a day. */
+  public List<Conversation> getRecentConversations() {
+    List<Conversation> recentConversations = new ArrayList<>();
+
+    Instant aDayBefore = Instant.now().minusSeconds(86400);
+
+    for (Conversation conversation : conversations) {
+      if(conversation.getCreationTime().isAfter(aDayBefore)){
```

Comment:
if( --> if ( 

(add space)

Reply:
Also, can you check if the conversations list is sorted? It might be beneficial for us to keep it sorted by timestamp that we can break early when we find a conversation older than a day. More scalable